src/components/Navbar.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC } from "react";
+import { NavLink, useNavigate } from "react-router-dom";
+import { Container, Nav, Navbar } from "react-bootstrap";
+import "../styles/Navbar.scss";
+import { useTypedSelector } from "../hooks/useTypedSelector";
+const logo = require("../images/logo.png");
+
+const NavBar: FC = () => {
+  const { isLogin } = useTypedSelector((store) => store);
+  const navigate = useNavigate();
+  const goToLink = (event: React.MouseEvent<HTMLAnchorElement>): void => {
```

Comment:
There is should be separate component with single responsibility of login button